### PR TITLE
WT-10608 Create testy dashboard

### PR DESCRIPTION
**Pull request by ruby-oujo**

### CODE REVIEW

**Review by tammybailey — APPROVED** [review]
This looks good to me! We can improve upon it once we have the IAM roles :)

File: services/scripts/testy-snapshot.sh [hunk 1]
```diff
@@ -71,10 +71,15 @@ main() {
 
     # Validate database. Update the snapshot status on success/failure.
     echo "Running validation script '$_validation_script' on volume '$_backup_volume_id'."
+    var=$(date +%s%3N)
```
Comment (Sean04) [inline services/scripts/testy-snapshot.sh:74]: Is there any reason why this variable doesn't follow the convention of the others using `local` and prefixing with an `_`?

Comment (ruby-oujo) [reply services/scripts/testy-snapshot.sh:74]: var is used in bash to store temporary variables, so I have just followed that here. 

We are using var to temporarily save the timestamp output from bash, and it always gets used right after it is updated, and won't be used anywhere else and becomes redundant immediately after.

Comment (Sean04) [reply services/scripts/testy-snapshot.sh:74]: Okay, I see, is it necessary then?

Comment (ruby-oujo) [reply services/scripts/testy-snapshot.sh:74]: Yup! I have to call it and save it somewhere before sending it into the logs. I cant call the bash command within the aws command unfortunately - or there may be a work around that I have not found.

Comment (Sean04) [reply services/scripts/testy-snapshot.sh:74]: Okay, that's fine then!

**Review by Sean04 — APPROVED** [review]
LGTM

**Pull request merged**